### PR TITLE
[FLINK-26737] Add CRD management in development doc

### PR DESCRIPTION
**Pull request by Aitozi**
Add CRD management in development doc

### CODE REVIEW

File: docs/content/docs/development/guide.md [hunk 1]
```diff
@@ -93,6 +93,22 @@ rest.port: 8081
...
 
+### Uninstalling the operator locally
+```bash
+helm uninstall flink-operator
+```
+
+### CRD Generating and Upgrading 
+
+By default, the CRD is generated by [Fabric8 CRDGenerator](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CRD-generator.md), when building from source.
+When installing flink-operator first time, the CRD will apply to the kubernetes cluster automatically. But it will not be removed or upgraded when re-install the flink-operator, referring to helm [doc](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/).
```
[the remark this answers is not in the document]
Comment (mbalassi) [reply docs/content/docs/development/guide.md:104]: Please update:
When installing flink-operator _for the_ first time...

**Review by mbalassi — COMMENTED** [review]
Thanks @Aitozi, I added some suggestions for rewording.

File: docs/content/docs/development/guide.md [hunk 2]
```diff
@@ -93,6 +93,22 @@ rest.port: 8081
...
 ```
 
+### Uninstalling the operator locally
+```bash
+helm uninstall flink-operator
+```
+
+### CRD Generating and Upgrading 
+
+By default, the CRD is generated by [Fabric8 CRDGenerator](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CRD-generator.md), when building from source.
```
[the remark this answers is not in the document]
Comment (Aitozi) [reply docs/content/docs/development/guide.md]: fixed

File: docs/content/docs/development/guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aitozi) [reply docs/content/docs/development/guide.md]: fixed

**Comment by Aitozi** [comment]
@mbalassi Thanks for your comments, I have fixed it. Please take a look again.

File: docs/content/docs/development/guide.md [hunk 3]
```diff
@@ -93,6 +93,22 @@ rest.port: 8081
...
 
+### Uninstalling the operator locally
+```bash
+helm uninstall flink-operator
+```
+
+### CRD Generating and Upgrading 
+
+By default, the CRD is generated by the [Fabric8 CRDGenerator](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CRD-generator.md), when building from source.
+When installing flink-operator for the first time, the CRD will apply to the kubernetes cluster automatically. But it will not be removed or upgraded when re-install the flink-operator, as described in the relevant helm [documentation](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/).
```
Comment (gyfora) [inline docs/content/docs/development/guide.md:104]: Should be: 
the CRD will **be applied** to the kubernetes
upgraded when **re-installing** the flink-operator

File: docs/content/docs/development/guide.md [hunk 4]
```diff
@@ -93,6 +93,22 @@ rest.port: 8081
 rest.address: localhost
 ```
 
+### Uninstalling the operator locally
+```bash
+helm uninstall flink-operator
+```
+
+### CRD Generating and Upgrading
```
Comment (gyfora) [inline docs/content/docs/development/guide.md:101]: Should be: **Generating and Upgrading the CRD**

**Review by gyfora — CHANGES_REQUESTED** [review]
added some minor grammar fixes otherwise looks good :)

File: docs/content/docs/development/guide.md (hunk 3 above)
Comment (Aitozi) [reply docs/content/docs/development/guide.md]: Fixed

File: docs/content/docs/development/guide.md (hunk 4 above)
Comment (Aitozi) [reply docs/content/docs/development/guide.md]: Fixed

**Pull request merged**